### ISSUE TITLE
Spell attributes

### ISSUE THREAD

**Issue by killerwife** [sidsukana/QSpellWork#2]
Can you please update with spell attributes that have been made known? (possibly look in TrinityCore code, they have more found than cmangos)

**Comment by sidsukana** [comment]
@killerwife Some attributes may be incorrect for the classics (you need to check them)
You can do PR with a modified Enums.xml for branch pre-tbc. Then I will add it directly to branche qsw-2.0.
Or give me link to updated Enums.xml :)

**Comment by killerwife** [comment]
Alright, I will, because I have several more confirmed on vengeance that I know are good, and that might help more people I think. Btw attributes were never changed to my knowledge, only added.

**Comment by sidsukana** [comment]
@killerwife I remind you that Enums are convenient to use because they are in Enums.xml file, and you can at any time change the attributes as you want. Also you can use enums constant name in scripting filter :) Ex: spell.Attributes & SPELL_ATTR_RANGED

**Comment by killerwife** [comment]
I know, but I dont want this info to be only for me :)

**Comment by killerwife** [comment]
https://gist.github.com/killerwife/e0c4d00d5868f42f86cb50f90a0e8d82
@sidsukana there you go, I added all stuff that is in Vengeance core and wasnt in QSW :) I only did so for stuff I am 100% sure of.

**Comment by sidsukana** [comment]
@killerwife it's for TBC?

**Comment by killerwife** [comment]
Yes. Should I discover more in the future, I will share them as well.

**Comment by killerwife** [comment]
https://gist.github.com/killerwife/6b6835bec633c248c57dae6d5602da6d @sidsukana latest TBC.xml enum

**Comment by killerwife** [comment]
https://gist.github.com/killerwife/ddbbbe6cd5abb271831d7cad4ed3b1c2 @sidsukana latest TBC.xml enum

**Comment by killerwife** [comment]
latest TBC.xml Enum - https://gist.github.com/killerwife/09381b3453fe896432a9f4146679b61a
